README.md

```diff
@@ -50,11 +50,12 @@ Get a pre-signed URL for a given operation name.
   - **parameters** - { Object } - parameters to pass to the operation
   - **urlTTL** - { Number } - URL expiration time in seconds
 
-#### putObject(context, { bucket, data, [key] })
+#### putObject(context, { bucket, data, [key] }, params)
 Adds an object to a bucket.
   - **bucket** - { String } - name of the bucket
-  - **data** - { Any } - object data; can be a String or any type that can be stringified
+  - **data** - { Buffer | Typed Array | Blob | String | ReadableStream } - object data
```

Comment:
does it actually support a stream? my understanding is put does not, but upload does?

Reply:
it does, according to the docs